DEVELOPING_OPAL/validate/src/test/scala/org/opalj/fpcf/ImmutabilityTests_closedWorld.scala

```diff
@@ -0,0 +1,74 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj.fpcf
+
+import java.net.URL
+import org.opalj.ai.domain.l1
+import org.opalj.ai.fpcf.properties.AIDomainFactoryKey
+import org.opalj.br.analyses.Project
+import org.opalj.br.fpcf.analyses.LazyL0CompileTimeConstancyAnalysis
+import org.opalj.br.fpcf.analyses.LazyStaticDataUsageAnalysis
+import org.opalj.tac.cg.TypeBasedPointsToCallGraphKey
+import org.opalj.tac.fpcf.analyses.escape.LazySimpleEscapeAnalysis
+import org.opalj.tac.fpcf.analyses.immutability.EagerL0FieldImmutabilityAnalysis
+import org.opalj.tac.fpcf.analyses.immutability.EagerClassImmutabilityAnalysis
+import org.opalj.tac.fpcf.analyses.immutability.EagerTypeImmutabilityAnalysis
+import org.opalj.tac.fpcf.analyses.immutability.field_assignability.EagerL2FieldAssignabilityAnalysis
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+
+/**
+ * Tests the immutability analysis with a closed world assumption.
+ *
+ * @author Tobias Roth
+ */
+class ImmutabilityTests_closedWorld extends PropertiesTest {
```

Comment:
I don't like this naming scheme. In Java/Scala underscores are not used, but CamelCase is.